### PR TITLE
Bring develop into line with alphagov/transition:master

### PR DESCRIPTION
**Pull request by lozette**
`develop` was 449 commits behind the upstream version. 

This PR brings us back into line with the original repo, without losing our custom features.

### CODE REVIEW

File: spec/javascripts/support/jasmine_helper.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 class HeadlessChromeJasmineConfigurer < JasmineSeleniumRunner::ConfigureJasmine
   def selenium_options
-    { options: GovukTest.headless_chrome_selenium_options }
+    args = %w[no-sandbox disable-dev-shm-usage disable-gpu]
+    Selenium::WebDriver::Chrome::Options.new(args: args)
```
Comment (Floppy) [inline spec/javascripts/support/jasmine_helper.rb]: I think you might need the `headless!` bit and return the same hash structure too

```
    args = %w[no-sandbox disable-dev-shm-usage disable-gpu]
    options = Selenium::WebDriver::Chrome::Options.new(args: args)
    options.headless!
    return { options: options }
```

File: Gemfile [hunk 2]
```diff
@@ -44,11 +45,15 @@ group :development do
 end
 
 group :test do
+  gem "capybara"
   gem "cucumber-rails", require: false
+  gem "cuprite"
   gem "database_cleaner"
   gem "factory_bot_rails"
   gem "launchy" # Primarily for save_and_open_page support in Capybara
+  gem "poltergeist"
```
Comment (Floppy) [inline Gemfile]: This might not be needed any more?

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
 class ApplicationController < ActionController::Base
   before_action :exclude_all_users_except_admins_during_maintenance
   before_action :authenticate
-  before_action :set_paper_trail_whodunnit
```
Comment (Floppy) [inline app/controllers/application_controller.rb]: Should this have gone away? `paper_trail` is still in use, I think?

**Review by Floppy — CHANGES_REQUESTED** [review]
One comment to remove poltergeist (I think that will be OK) and one question around removal of a line, though that might have been something we did months ago in which case, fine.

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (lozette) [reply app/controllers/application_controller.rb]: I can't actually find that method, and I can't find it in the alphagov transition version either so I'm not sure what it refers to 🤔

Comment (lozette) [reply app/controllers/application_controller.rb]: Oh wait it's a papertrail method included in the gem!

**Review by Floppy — APPROVED** [review]

**Pull request merged**